### DISCUSSION TITLE
服务器用完内存后，会突然发生磁盘 IO 爆满。

### DISCUSSION THREAD

**Discussion by AkarinLiu** [1Panel-dev/1Panel#5473] [Q&A]
### 联系方式

akarinliu@outlook.com

### 1Panel 版本

v1.10.10

### 问题描述

服务器在运行一段时间后，内存占用会逐渐增加，直到用完内存后使用硬盘的 swap，导致服务器宕机，怀疑有内存泄露的 Bug。

### 重现步骤

1.安装 wordpress、mariadb 等应用
2. 创建三个数据库
3. 使用源码部署 Flarum 和 Yourls

### 期待的正确结果

内存占用不持续升高，服务器稳定运行。

### 附加信息

![Screenshot_2024-06-14-18-45-02-442_com microsoft emmx-edit](https://github.com/1Panel-dev/1Panel/assets/58081491/ff8ec34c-1614-40af-8c2f-c4bd0db2493c)
![Uploading Screenshot_2024-06-14-18-43-04-213_com.microsoft.emmx-edit.jpg…]()

**Comment by AkarinLiu** [comment]
补充一张内存占用趋势图：
![Screenshot_2024-06-15-04-39-00-157_com microsoft emmx-edit](https://github.com/1Panel-dev/1Panel/assets/58081491/f5cf867e-38de-4e9e-9725-19b925fb3c2b)

**Comment by wanghe-fit2cloud** [comment]
建议先自行分析一下各个服务的内存占用情况。

**Comment by YYJeffrey** [comment]
Mark 一下，遇到了相同的情况，监测到发生磁盘 Read 峰值达到 100 MByte/s，当时未进行任何主动操作，各容器日志也无异样。
<img width="531" alt="image" src="https://github.com/user-attachments/assets/4926035a-fec9-418b-b89d-28c05e9458a8">

**Comment by KangWang42** [comment]
同样遇到这个情况
